qualtran/bloqs/chemistry/trotter.ipynb

```diff
@@ -0,0 +1,752 @@
+    "        return None\n",
+    "    if isinstance(bloq, CirqGateAsBloq):\n",
+    "        if isinstance(bloq.gate, cirq_ft.algos.And) and (len(bloq.gate.cv) == 2):\n",
+    "            return And(cv1=and_cv0, cv2=and_cv1, adjoint=bloq.gate.adjoint)\n",
+    "        if \"QROM\" in bloq.pretty_name():\n",
+    "            bloq.gate.__class__.__repr__ = custom_repr\n",
+    "        if isinstance(bloq.gate, cirq_cliffords_ignore):\n",
+    "            return None\n",
+    "    if isinstance(bloq, Join):\n",
+    "        return None\n",
+    "    if isinstance(bloq, Allocate):\n",
+    "        return None\n",
+    "    if isinstance(bloq, Free):\n",
+    "        return None\n",
+    "    if isinstance(bloq, RotationBloq):\n",
+    "        return attrs.evolve(bloq, angle=phi)\n",
+    "    return bloq"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qualtran.bloqs.chemistry.trotter import PolynmomialEvaluation \n",
+    "poly_eval = PolynmomialEvaluation(14, 15, 24)\n",
+    "graph, sigma = get_bloq_counts_graph(poly_eval, generalizer=generalize)\n",
+    "GraphvizCounts(graph).get_svg()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "paper_cost = 4 * (3*15**2 + 45)\n",
+    "print(f\"qualtran cost = {poly_eval.t_complexity().t} vs paper_cost = {paper_cost} T gates\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qualtran.bloqs.chemistry.trotter import NewtonRaphson\n",
+    "nr = NewtonRaphson(14, 15, 24)\n",
+    "graph, sigma = get_bloq_counts_graph(nr, generalizer=generalize)\n",
+    "GraphvizCounts(graph).get_svg()\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "paper_cost = 4 * (2136 - 3*15**2 - 45)\n",
+    "print(f\"qualtran cost = {nr.t_complexity().t} vs paper cost = {paper_cost} T gates\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qualtran.resource_counting import get_bloq_counts_graph\n",
+    "from qualtran.drawing import GraphvizCounts\n",
+    "from qualtran.bloqs.chemistry.trotter import KineticEnergy \n",
+    "ke = KineticEnergy(4, 21)\n",
+    "graph, sigma = get_bloq_counts_graph(ke, generalizer=generalize)\n",
+    "GraphvizCounts(graph).get_svg()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qualtran.resource_counting import get_bloq_counts_graph\n",
+    "from qualtran.drawing import GraphvizCounts\n",
+    "from qualtran.bloqs.chemistry.trotter import PotentialEnergy \n",
+    "pe = PotentialEnergy(4, 21)\n",
+    "graph, sigma = get_bloq_counts_graph(pe, generalizer=generalize)\n",
```

Comment:
The nodes in these graphs are very wide. Is there an issue open to track the issue of making these prettier? If yes: please specifically note this cell in this notebook

Reply:
There isn't AFAIK. I'll open one.